projects/m1/034-integer-to-ordinal/js/index.js

```diff
@@ -0,0 +1,72 @@
+//1) array solution.
+function getNumberFromUser (myNum) {
+  num = new Array(); //Created an array which contain twelve elements with twelve value,in this case every element contain a string 
+  num[1] = 'First';
+  num[2] = "Second";
+  num[3] = "Third";
+  num[4] = 'Fourth';
+  num[5] = "Fifth";
+  num[6] = "Sixth";
+  num[7] = 'Seventh';
+  num[8] = "eighth";
+  num[9] = "Nineth";
+  num[10] = 'Tenth';
+  num[11] = "Eleventh";
+  num[12] = "Twelfth";
+
+  return num[myNum];
```

Comment:
**Not a good solution** to use with arrays. 
In case you want to use arrays, **immediately declare** it as follows:
`numbers = ["First", "Second", "etc]´
Then the single element is accessed as follows:
`numbers[myNum - 1]`